homeassistant/components/netatmo/climate.py

```diff
@@ -168,6 +178,23 @@ def __init__(self, data, room_id):
         if self._module_type == NA_THERM:
             self._operation_list.append(HVAC_MODE_OFF)
 
+        self._unique_id = f"{self._room_id}-{self._module_type}"
+
+    @property
+    def device_info(self):
+        """Return the device info for the thermostat/valve."""
+        return {
+            "identifiers": {(DOMAIN, self._room_id)},
```

Comment:
Is there always only 1 device per room ?

Reply:
And can room id change  ?

Reply:
Netatmo exposes not individual devices but rooms. So even if one room contains two or more valves you can only control on room level.
As far as I'm aware the room id doesn't change.